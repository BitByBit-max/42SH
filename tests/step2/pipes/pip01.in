echo "This is the start of the pipeline" | sed 's/start/end/' | grep end
