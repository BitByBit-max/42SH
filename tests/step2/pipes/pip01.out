This is the end of the pipeline
